nimbleparse/src/main.rs

```diff
@@ -62,8 +62,8 @@ fn main() {
         .optopt(
             "r",
             "recoverer",
-            "Recoverer to be used (default: mf)",
-            "cpctplus|mf|none"
+            "Recoverer to be used (default: cpctplus)",
+            "mf|none"
```

Comment:
I think  you wanted to remove `mf` here and keep `cpctplus`.

Reply:
Good spot!

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/117/commits/7316e6fe583d0b642501920920625cefead6eafb.